app/controllers/concerns/blacklight/catalog.rb

```diff
@@ -12,6 +12,7 @@ module Blacklight::Catalog
   # The following code is executed when someone includes blacklight::catalog in their
   # own controller.
   included do
+    helper BlacklightHelper
```

Comment:
Doesn't the blacklight-provided layout require these? Should this go on `Blacklight::Controller`?

Reply:
Yes, good catch. I can make that change.